### ISSUE TITLE
URLs of the plugin

### ISSUE THREAD

**Issue by carstingaxion** [GatherPress/gatherpress#744]
### Is your enhancement related to a problem? Please describe.

A visual overview of all public URLs generated by GatherPress, to help fixing some bugs and start writing some docs about that.

### Bugs related to URLs and permalinks

- #682 
- #722 
- #704 
- #603 

### Actual URL scheme

```
 - events
    - %EVENT_TITLE%
       - feed
          - comments
 - venues
    - %VENUE_TITLE%
 - topics
    - %TOPIC_TITLE%
```

### Suggested URL scheme

```
 - %PT_ROOT_EVENTS%
    - ical 
    - %EVENT_TITLE%-%EVENT_START%-%VENUE_TITLE%
       - ical 
       - google-calendar
       - …
       - feed
          - rsvp
 - %PT_ROOT_VENUES%
    - %VENUE_TITLE%
 - %TAX_ROOT_TOPICS%
    - %TOPIC_TITLE%
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by mauteri** [comment]
Closing this. Nothing left on it that isn't tracked better elsewhere.

**The bug checklist is mostly discharged.** #722, #704, #603, and #901 are all closed. The two remaining aren't waiting on this tracker:

- #826 (rewrite and permastruct tags) is in 0.37.0, and its tag inventory was expanded today with everything the 0.34.0 structured address meta makes possible.
- #682 (permalink generation scheme) has an open proposal to close it, on the basis that core's job is providing the tags rather than shipping an opinionated scheme.

**The "Actual URL scheme" table has gone stale**, which is the main reason not to carry it forward. It predates the calendar feed work in 0.34.0 and is missing the sitewide, venue, and topic feeds entirely. The current, accurate map lives in [`docs/user/calendar-feeds.md`](../blob/develop/docs/user/calendar-feeds.md):

| Feed | URL |
| --- | --- |
| All events on the site | `example.org/feed/ical` |
| The events archive | `example.org/event/feed/ical` |
| Events at one venue | `example.org/venue/my-venue/feed/ical` |
| Events in one topic | `example.org/topic/my-topic/feed/ical` |

**The "Suggested URL scheme"** is superseded by #826's tag inventory, which is more complete and grounded in data we actually store.

**And the docs half happened**, just distributed rather than as one page: user-facing feed URLs in `docs/user/calendar-feeds.md`, and the Endpoint API with a worked example in `docs/developer/custom-url-endpoints/README.md`.

@carstingaxion, this did its job. Six URL bugs got tracked and four are fixed, and mapping the scheme out early is a large part of why the endpoint subsystem and the feeds ended up coherent rather than accreted.
